### DISCUSSION TITLE
Resume scan on power-up

### DISCUSSION THREAD

**Discussion by oisact** [egzumer/uv-k5-firmware-custom#242] [Ideas]
I think the radio should resume scanning on power-on if it was scanning when powered off. Most other radios function this way. Thanks.

**Comment by ErikS-web** [comment]
None of my HAM-radio's do that !

**Reply by oisact** [reply]
I have multiple Yaesu rigs that do.

**Comment by egzumer** [comment]
This doesn't make sense whatsoever. If you want to scan, scan do not turn of the radio.

**Comment by AubsUK** [comment]
I have just implemented this on the first version of firmware and CHIRP driver over on my repo - https://github.com/AubsUK/uv-k5-firmware-custom

The current main changes to Egzumer's firmware are:

* The two ScanLists have been removed and replaced with 10 ScanLists (0-9)
* When scanning, the ScanLists now show along the top of the screen
* Toggling ScanLists on/off is as simple as pressing the number
* Memory channels can now be excluded (LockOut/Skip) from scanning
* Scanning can be set to automatically start when the radio is turned on
* All the changes can be managed through CHIRP

You WILL need a custom driver to use CHIRP.  All instructions provided.

Open a discussion there if you have any questions or suggestions.
